client/src/components/reservation/reservation.js

```diff
@@ -9,8 +14,18 @@ const Reservation = () => {
   const [endDate, setEndDate] = useState(
     startDate.getTime() + 60 * 60 * 1000,
   );
+  const dispatch = useDispatch();
+  const reservationCreated = useSelector(
```

Comment:
Please create new file for selectors in store folder `export const getCreatedReservation = state => state.reservation?.reservation_created,`
 and import selector here:


```suggestion
  const reservationCreated = useSelector(state => getCreatedReservation(state));
```